### ISSUE TITLE
Growth Disabling

### ISSUE THREAD

**Issue by x1p** [BentoBoxWorld/BSkyBlock#134]
**Description**
Disable Growth does not work quite strict.

**Steps to reproduce the behavior:**
As long as the player is not been online that day, the growth is stopped. Once the player has been online, the growth continues even though the player is not online anymore.

**Expected behavior**
Once a player is offline, growth should be frozen.

**Screenshots**

**Server Information:**

[Please complete the following information:]
 - Database being used (Flat file, MySQL, Mongo): JSON
 - OS: Linux
 - Java Version: 1.8.0_212
 - BentoBox version: 1.5.0 build 1118
 - Addons installed?
Running PAPER 1.13.2.
BentoBox version: 1.5.0-SNAPSHOT-b1118
Loaded Game Worlds:
bskyblock_world (BSkyBlock): Overworld, Nether, End
Loaded Addons:
Biomes 1.5.0.0-SNAPSHOT-b86
BSkyBlock 1.5.0-b313
Challenges 0.7.5-SNAPSHOT-b244
Level 1.5.0-SNAPSHOT-b203
Limits 0.2.0-SNAPSHOT-b111
MagicCobblestoneGenerator 1.5.0.1-b45

**Additional context**
I have a custom growth disable plugin from Askyblock. I'll be happy to share if it helps.

**Comment by tastybento** [comment]
Yes, this is a bug. It looks like actually this class was not finished in terms of coding because currently if it is set, then no crops will grow at all anywhere.
